operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -113,6 +116,61 @@ protected Status createStatus() {
                 })));
     }
 
+    @Test
+    public void testSuccessfulReconcileWithSelectorLabel(VertxTestContext context)  {
+        MetricsProvider metrics = createCleanMetricsProvider();
+
+        AbstractWatchableStatusedResourceOperator resourceOperator = resourceOperatorWithExistingResource();
+
+        Labels nsLabel = Labels.fromMap(Collections.singletonMap("nsLabelKey", "nsLabelValue"));
```

Comment:
`nsLabels` is something I would read s namespace ... but this looks like a resource labels? Should we give it some better name? Same for the values I guess?